src/pwa/pages/maps.tsx

```diff
@@ -1,6 +1,40 @@
+import { Grid, Typography } from '@mui/material';
+import GoogleMapReact from 'google-map-react';
+import { useEffect, useState } from "react";
 const Map = () => {
+    const [currentGeoPosition, setCurrentGeoPosition] = useState<any>();
+
+    const getCurrentPosition = () => {
+        navigator.geolocation.getCurrentPosition((position) => {
+            console.info(position)
+            setCurrentGeoPosition(position)
+        });
+    }
+
+    useEffect(() => {
+        getCurrentPosition();
+    }, []);
+
     return (
-        <h4>Map</h4>
```

Comment:
I would add a small section above map, showing current GPS latitude and longitude.

Reply:
I added the current location. help me verify it